includes/Admin.php

```diff
@@ -78,6 +78,8 @@ public function init() {
 		add_action( 'wp_ajax_mark_onboarding_done', array( $this, 'mark_onboarding_done' ) );
 		add_action( 'wp_ajax_nopriv_mark_onboarding_done', array( $this, 'mark_onboarding_done' ) );
 
+		add_action( 'wp_ajax_external_get_logs', array( $this, 'external_get_logs' ) );
```

Comment:
Just to be safe, we can use something less generic as the ajax name to avoid conflicts.